src/main/java/sg/ncl/MainController.java

```diff
@@ -144,39 +144,121 @@ public String tools() {
+//            throw new FuturePlanDownloadException("IOError writing file to output stream");
+//        } finally {
+//            if (stream != null) {
+//                stream.close();
+//            }
+//        }
+//    }
+
+    @RequestMapping(value="/orderform/download", method=RequestMethod.GET)
+    public void OrderForm_v1Download(HttpServletResponse response) throws OrderFormDownloadException, IOException {
+        InputStream stream = null;
+        response.setContentType("application/pdf");
+        try {
+            stream = getClass().getClassLoader().getResourceAsStream("downloads/order_form.pdf");
+            response.setContentType("application/force-download");
+            response.setHeader("Content-Disposition", "attachment; filename=order_form.pdf");
+            IOUtils.copy(stream, response.getOutputStream());
+            response.flushBuffer();
+        } catch (IOException ex) {
+            logger.info("Error writing file to output stream.");
+            throw new OrderFormDownloadException("IOError writing file to output stream");
+        } finally {
+            if (stream != null) {
+                stream.close();
+            }
+        }
+    }
+
+    @RequestMapping(value="/SubscriptionAgreement/download", method=RequestMethod.GET)
+    public void SubscriptionAgreementDownload(HttpServletResponse response) throws MasterSubscriptionAgreementDownloadException, IOException {
         InputStream stream = null;
         response.setContentType("application/pdf");
         try {
-            stream = getClass().getClassLoader().getResourceAsStream("downloads/future_plan.pdf");
+            stream = getClass().getClassLoader().getResourceAsStream("downloads/SubscriptionAgreement.pdf");
             response.setContentType("application/force-download");
-            response.setHeader("Content-Disposition", "attachment; filename=future_plan.pdf");
+            response.setHeader("Content-Disposition", "attachment; filename=SubscriptionAgreement.pdf");
             IOUtils.copy(stream, response.getOutputStream());
             response.flushBuffer();
-        } catch (Exception ex) {
+        } catch (IOException ex) {
             logger.info("Error writing file to output stream.");
-            throw new FuturePlanDownloadException("IOError writing file to output stream");
+            throw new MasterSubscriptionAgreementDownloadException("IOError writing file to output stream");
         } finally {
             if (stream != null) {
                 stream.close();
             }
         }
     }
 
-    @RequestMapping(value="/orderform/download", method=RequestMethod.GET)
-    public void OrderForm_v1Download(HttpServletResponse response) throws OrderFormDownloadException, IOException {
+    @RequestMapping(value="/UsagePolicy/download", method=RequestMethod.GET)
+    public void UsagePolicyDownloadDownload(HttpServletResponse response) throws UsagePolicyDownloadException, IOException {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this method to throw at most one checked exception instead of: sg.ncl.exceptions.UsagePolicyDownloadException, java.io.IOException [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1160)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00100)